package.json

```diff
@@ -48,9 +42,8 @@
   },
   "license": "ISC",
   "lint-staged": {
-    "*.{js,jsx,vue,ts,tsx}": [
-      "vue-cli-service lint",
-      "git add"
```

Comment:
Vue cli generate this part without this line, so it looks like it can be omitted 🤔